lib/CodeGen/Address.h

```diff
@@ -25,11 +25,41 @@ namespace CodeGen {
 class Address {
   llvm::Value *Pointer;
   CharUnits Alignment;
+
+private:
+  bool _containMMSafePtr;
```

Comment:
This data member will be uninitialized since not all paths through the constructor body initialize it. Please consider initializing it to prevent uninitialized memory bugs.

Reply:
Similar observation for the data members originalPointerTy and rawPointerTy. Consider initializing them to nullptr.

Reply:
Thanks for the remind. I was aware of the uninitialization. I didn't bother to initialize them because until now the functions that use them are actually not called by any other functions. But you're right that it's good programming practice to always initialize the variables. I'll fix them soon.